src/pylero/document.py

```diff
@@ -454,7 +451,13 @@ def get_work_items(
             ).uri
         else:
             parent_uri = None
-        p_fields = _WorkItem._convert_obj_fields_to_polarion(fields)
+        p_fields = list()
+        for wi_type in _WorkItem.get_defined_work_item_types():
+            if not wi_type._got_custom_fields:
+                wi_type.get_custom_fields(self.project_id)
+            filtered_fields = [f for f in fields if f in wi_type._cls_suds_map.keys()]
+            p_fields.extend(wi_type._convert_obj_fields_to_polarion(filtered_fields))
+        p_fields = list(set(p_fields))
         suds_wi = self.session.tracker_client.service.getModuleWorkItems(
```

Comment:
I have refrained from changing the method `_convert_obj_fields_to_polarion` because of the nifty check it performs
for custom fields.
Piping through any fields staring with 'customFields.*' feels like violating the abstaction of treating normal and custom fields the same as members of the work item, cf. this comment https://github.com/RedHatQE/pylero/pull/137#issuecomment-1531417303.

Therefore I propose to do the whole check here in the `Document` class which makes sure that calling the inerhited method `_convert_obj_fields_to_polarion` will throw no error :)